invoicing/sap/fetch/process.py

```diff
@@ -79,7 +79,7 @@ def process_payment_data(  # noqa: C901
                 invoice_number=invoice_number
             )
         except ApartmentInstallment.DoesNotExist:
-            errors.append(
+            logger.error(
                 f"{line_number}: ApartmentInstallment with invoice number "
                 f'"{invoice_number}" does not exist.'
             )
```

Comment:
Btw... it would be better if the log message and its arguments were separated in the logger call, because then Sentry is able to group all of these errors to same error rather than creating several different errors for each instance. I.e.
```suggestion
            logger.error(
                "ApartmentInstallment with invoice number %s does not exist (line %s)",
                invoice_number,
                line_number,
            )
```